src/models/tools/image/image-content.ts

```diff
@@ -1,83 +1,52 @@
 import { types, Instance, SnapshotOut } from "mobx-state-tree";
-import { createChange, ImageToolChange } from "./image-change";
-import { exportImageTileSpec, importImageTileSpec, isImageTileImportSpec } from "./image-import-export";
+import { exportImageTileSpec, isLegacyImageTileImport, convertLegacyImageTile } from "./image-import-export";
 import { ITileExportOptions, IDefaultContentOptions } from "../tool-content-info";
 import { ToolContentModel } from "../tool-types";
 import { isPlaceholderImage } from "../../../utilities/image-utils";
-import { safeJsonParse } from "../../../utilities/js-utils";
 import placeholderImage from "../../../assets/image_placeholder.png";
 
 export const kImageToolID = "Image";
 
 // This is only used directly by tests
 export function defaultImageContent(options?: IDefaultContentOptions) {
-  const change = createChange(options?.url || placeholderImage);
-  return ImageContentModel.create({ changes: [change] });
+  return ImageContentModel.create({url: options?.url || placeholderImage});
 }
 
 export const ImageContentModel = ToolContentModel
   .named("ImageTool")
   .props({
     type: types.optional(types.literal(kImageToolID), kImageToolID),
-    changes: types.array(types.string)
+    changes: types.maybe(types.array(types.string)), //keeping for legacy documents
```

Comment:
Do we really need to keep this for legacy documents? I would have thought that making sure they got migrated successfully would be sufficient and we wouldn't want to include `changes` in current models.

Reply:
I needed to keep it when reading in the legacy document.